### PR TITLE
Anton/us 868 fix waiters

### PR DESCRIPTION
**Pull request by romanukhaav**

### CODE REVIEW

File: src/test/java/rv/atqc/gttesting/archex/PlacesPage.java [hunk 1]
```diff
@@ -148,7 +148,7 @@ public PlacesPage closeResults(){
 	
 	public PlacesPage lookForItem(String searchWord){
 		driver.get(PLACES_PAGE_URL);
-		waitForVisibilityOfElement(featuredPlace,MAX_WAIT_TIME);
+		waitForVisibilityOfElement(searchField,MAX_WAIT_TIME);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/PlacesPage.java:150]: Why you changed it?

File: src/test/java/rv/atqc/gttesting/archex/PlacesPage.java [hunk 2]
```diff
@@ -157,10 +157,12 @@ public PlacesPage lookForItem(String searchWord){
 	}
 	
 	public void setFilters(boolean[] filter){
+		waitForVisibilityOfElement(placesButton, MAX_WAIT_TIME);
 		openCategoryPlaces();
 		for (int i=0; i<5; i++){
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/PlacesPage.java:161]: why 5?

**Review by dmatsev — APPROVED** [review]
with questions

File: src/test/java/rv/atqc/gttesting/archex/PlacesPage.java (hunk 2 above)
Comment (romanukhaav) [reply src/test/java/rv/atqc/gttesting/archex/PlacesPage.java:161]: because we have only 5 types of places in filterList Array from 0 to 4

**Pull request merged**